lib/yggdrasil/GuardianErrorHandler.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.GuardianErrorHandler do
+  @behaviour Guardian.Plug.ErrorHandler
+
+  import Plug.Conn
+
+  def unauthenticated(conn, _params) do
+    conn
+      |> send_resp(:unauthorized, "")
```

Comment:
Ah that makes sense. Might want to leave a comment saying that you gotta use :unauthorized so the user will get a 401.